src/online-members/utils.js

```diff
@@ -21,13 +21,22 @@ async function makeApiCall(
 
 async function getMembersData() {
   let membersList = null;
+  const memberObject = {};
   const membersRequest = await makeApiCall(RDS_API_MEMBERS);
   if (membersRequest.status === 200) {
     membersList = await membersRequest.json();
     membersList = membersList.members;
     membersList = membersList.filter((member) => !member.incompleteUserDetails);
+    membersList.forEach((member) => {
+      if (!member.incompleteUserDetails) {
+        memberObject[`${member.username}`] = {
+          isOnline: false,
+          ...member,
+        };
+      }
```

Comment:
Nit: By writing the error/failing condition in If block, we can unwarp the main code out of the guard block and focus on _why_ something failed or was not allowed.

```suggestion
      if  (member.incompleteUserDetails) {
        console.error('Member with incomplete detail');
        return;
      }
      
      memberObject[`${member.username}`] = {
        isOnline: false,
        ...member,
      };
```